### ISSUE TITLE
[Bug Report] Intense Lag When Setting Custom Status With Nitro Emoji

### ISSUE THREAD

**Issue by FortNbreak** [riolubruh/YABDP4Nitro#169]
I lag insanely bad when setting a custom status with a nitro emoji.

**Comment by FortNbreak** [comment]
also i tried restarting discord, removing all plugins except for YABDP4Nitro, and ZerosPluginLibrary

**Comment by riolubruh** [comment]
What the?? You're not supposed to be able to set a custom status with a nitro emoji...! How the hell did you do that?

**Comment by FortNbreak** [comment]
![Screenshot 2023-11-20 144013](https://github.com/riolubruh/YABDP4Nitro/assets/98566516/c66b21c1-3201-4b8a-9f24-c98e496ab9da)

**Comment by FortNbreak** [comment]
bug i guess lol

**Comment by FortNbreak** [comment]
idk if it is just client side or if other YABDP4Nitro users can see it but on my alt, i could not see it but on my main, i could

**Comment by riolubruh** [comment]
No, looks like this is a client side bug. The API still rejects nitro emoji in the status, but the client shows it anyways.
